docs/src/Theory/Common/SurfaceFluxes.md

```diff
@@ -1,105 +1,283 @@
+```@meta
+CurrentModule = ClimateMachine.SurfaceFluxes
+```
+
+This page contains a description of the SurfaceFluxes module, based on the Nishizawa surface-flux scheme.
 
-## `Byun1990`
+> Nishizawa, S., & Kitamura, Y. ( 2018). A surface flux scheme based on the Monin‐Obukhov similarity for finite volume models. _Journal of Advances in Modeling Earth Systems_, 10, 3159– 3175. [https://doi.org/10.1029/2018MS001534](https://doi.org/10.1029/2018MS001534)
+> <em>Internally abbreviated as NK</em>    
 
-Compute surface fluxes using the approach in Byun (1990).
+> Gryanik, V., Lupkes, C., Grachev, A. & Sidorenko, D. (2020). New modified and extended stability functions for the stable boundary layer based on SHEBA and parametrizations of bulk transfer coefficients for climate models. _Journal of the Atmospheric Sciences_, 10.1175/JAS-D-19-0255.1. [https://doi.org/10.1175/JAS-D-19-0255.1](https://doi.org/10.1175/JAS-D-19-0255.1). (Early Online Release) 
+> <em>Internally abbreviated as GLGS</em>
 
-### Plots
+>Hill, R.J., 1989. Implications of Monin–Obukhov Similarity Theory for Scalar Quantities. _J. Atmos. Sci.,_  **46**, 2236–2244, [https://doi.org/10.1175/1520-0469(1989)046<2236:IOMSTF>2.0.CO;2](https://doi.org/10.1175/1520-0469(1989)046%3C2236:IOMSTF%3E2.0.CO;2) 
 
-```@example byun1990
-using ClimateMachine.SurfaceFluxes.Byun1990
-using Plots, LaTeXStrings
-using CLIMAParameters
-struct EarthParameterSet <: AbstractEarthParameterSet end
-param_set = EarthParameterSet()
+### Parameters
+* $L_{MO}$ = Monin-Obukhov length
+* $X_{\star}$ = Scale parameter for variable $X$
+* $X_{sfc}$ = Surface value of variable $X$ 
+* $\kappa$ = von-Karman coefficient (typically $0.4$)
+* $\mathrm{D_T} = \nu/Pr_{T}$ is the diffusivity
+*  $Pr_{T}$ the neutral limit turbulent Prandtl number (typically $\frac{1}{3}$)
+* $\nu$ the kinematic viscosity
+* $w$, the wall normal velocity component
+* $\Delta z$ = Model reference altitude
+* $\delta_{\chi_{i}}$ a measure of mixing efficiency for $i$th tracer 
+* $\zeta = z/L_{MO}$ similarity variable
+* $\nu/u_{\star}$ Viscous lengthscale
 
-FT = Float64
-Ri_range = range(FT(-1.2), stop = FT(0.24), length = 100)
-scales = FT[50, 200, 600, 1000, 10_000]
+### Fluxes
 
-z_0 = FT(1.0)
-γ_m, γ_h = FT(15.0), FT(9.0)
-β_m, β_h = FT(4.8), FT(7.8)
-Pr_0 = FT(0.74)
+* Momentum fluxes due to boundary shear
+```math 
+-\rho \tau_{j3}|_{\mathrm{sfc}} = \rho \overline{w^{\prime}u_{j}^{\prime}}|_{\mathrm{sfc}} 
+``` where $u$ represents the surface parallel velocity component, for $j = 1,2$
 
-plot(Ri_range,
-     [Byun1990.compute_exchange_coefficients(param_set, Ri, scale * z_0, z_0,
-                                             γ_m, γ_h, β_m, β_h, Pr_0)[1]
-        for Ri in Ri_range, scale in scales],
-     xlabel = "Bulk Richardson number (Ri_b)",
-     ylabel = "Drag coefficient",
-     title = "Momentum exchange coefficient",
-     labels = scales, legendtitle=L"z/z_0")
+*  Sum of specific enthalpy and moist static energy flux
```

Comment:
This is just the moist static energy flux (MSE contains specific enthalpy plus potential energy)